src/main/java/com/puppycrawl/tools/checkstyle/DefaultConfiguration.java

```diff
@@ -36,10 +38,9 @@ public final class DefaultConfiguration implements Configuration {
 
     /** A unique serial version identifier. */
     private static final long serialVersionUID = 1157875385356127169L;
-
     /** Constant for optimization. */
-    private static final Configuration[] EMPTY_CONFIGURATION_ARRAY = new Configuration[0];
-
+    private static final @ArrayLen(0) Configuration[] EMPTY_CONFIGURATION_ARRAY =
```

Comment:
Put annotation above, not on same line

Reply:
@romani putting annotation above giving me violation in `checker-formatter` that it should be either before or after of the `private static final `

Reply:
Please put it before

Reply:
done!

Reply:
> @romani putting annotation above giving me violation in `checker-formatter` that it should be before type and after modifiers.